backend/src/Squidex/appsettings.json

```diff
@@ -1,4 +1,9 @@
 ﻿{
+  "devMode": {
+    "enable": true,
```

Comment:
We cannot set this to true, it would be a breaking change.

Reply:
this property will be removed.

Reply:
If we don't need it than we can also hard code the port in my opinion.